### ISSUE TITLE
Segmentation fault On startup, Before the gui show's up.

### ISSUE THREAD

**Issue by Meatbag93** [audacity/audacity#3969]
### Bug description

Hi,
I have switched to pulseaudio from pipewire, then started getting this problem. Then I Switched back to pipewire, thinking it would solve the problem. But still.

### Steps to reproduce

- Just launch audacity from anywhere (you need to do it from somewhere you can see terminal output)

### Expected behavior

Audacity should launch normally.

### Actual behavior

Instead I just get `Segmentation fault (core dumped)`

### Audacity Version

stable version (from a Linux repository)

### Operating system

Linux

### Additional context

Helpful information:
- Audacity v3.1.3
- Kernel Linux 5.17.5-300.fc36.x86_64
- Distribution Fedora Linux 36 (Workstation Edition)

Full command line output:

```

$ audacity
ALSA lib pcm_dsnoop.c:601:(snd_pcm_dsnoop_open) unable to open slave
ALSA lib pcm_dmix.c:1032:(snd_pcm_dmix_open) unable to open slave
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib pcm_dmix.c:1032:(snd_pcm_dmix_open) unable to open slave
Segmentation fault (core dumped)

```

Happy to answer any more questions.

**Comment by Meatbag93** [comment]
Update: I've discovered that audacity run's fine when ran as sudo. It might be a problem with some config in my side and not audacity itself, but I'm still not sure what's wrong.

**Comment by Meatbag93** [comment]
The issue seems to have been fixed when I installed jack audio connection kit. I have discovered that by complete chance, with the unhelpful errors I'm getting there was no way I'd know.

**Comment by insaner** [comment]
Based on this answer, the solution for me was to (re)install:
`pipewire-jack-audio-connection-kit `

dnf said that it was already installed but not at the latest version, but when I tried to upgrade, it refused. Just issuing "install" for the package did the trick.

(I tried gdb backtrace on the segfault which said it was a problem in: `client_node_transport () at /usr/lib64/pipewire-0.3/jack/libjack.so.0` from `client_node_demarshal_transport.lto_priv () at /usr/lib64/pipewire-0.3/libpipewire-module-client-node.so`... in case anyone lands here from a search engine)

Thank you!
